plugins/basic/js/shotgun/constants.js

```diff
@@ -36,14 +36,14 @@ sg_constants.product_info = {
     // Photoshop
     PHSP: {
         display_name: "Photoshop",
-        tk_engine_name: "tk-photoshop",
+        tk_engine_name: "tk-photoshopcc",
```

Comment:
upgraded javascript defaults - but note that these are unused in the python code.